src/cryptsetup/cryptsetup.c

```diff
@@ -565,11 +584,96 @@ static char *friendly_disk_name(const char *src, const char *vol) {
         return name_buffer;
 }
 
+static PasswordRegisteredType check_registered_passwords(struct crypt_device *cd) {
+
+        int slots = 0, slot_max;
+        PasswordRegisteredType r = _PASSWORD_REGISTERED_TYPE_INVALID;
```

Comment:
We reserve `r` for `int r`. Please call this something more meaningful, e.g. `type`.

Reply:
Ok.